inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.h

```diff
@@ -29,27 +32,44 @@ class MKLDNNPadNode : public MKLDNNNode {
         SYMMETRIC = 3
     };
 
-    void padConstantOrEdge(const float *srcData, float* dstData,
-                           InferenceEngine::SizeVector srcDims, InferenceEngine::SizeVector dstDims,
-                           const bool isEdge = false);
-    void padReflectOrSymmetric(const float *srcData, float* dstData,
-                               InferenceEngine::SizeVector srcDims, InferenceEngine::SizeVector dstDims,
-                               const bool isSymmetric = false);
+    void padConstant();
```

Comment:
It is better to place such auxiliary functions, which are not a part of the class interface, to the private section.